pytest_fixtures/core/contenthosts.py

```diff
@@ -237,6 +237,9 @@ def module_container_contenthost(request, module_target_sat, module_org, module_
         assert host.execute('systemctl enable --now podman').status == 0, (
             'Start of podman service failed'
         )
+        assert host.execute('yum -y install expect').status == 0, (
+            'Expect installation failed unexpectedly'
```

Comment:
I'd avoid using custom messages that don't provide more information than you'd get with the normal assertion.
In this case, the command failing already tells us that the installation failed. Since we're asserting that the status is 0, we're expecting it to pass. That brings us to what the custom assertion fail message tells us.
However, the normal assertion would also give us the actual status code; so more information than the custom exception.